pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -254,7 +254,8 @@ func (rc *reconciler) reconcile() {
 				nodes := rc.actualStateOfWorld.GetNodesForVolume(volumeToAttach.VolumeName)
 				if len(nodes) > 0 {
 					if !volumeToAttach.MultiAttachErrorReported {
-						simpleMsg, detailedMsg := volumeToAttach.GenerateMsg("Multi-Attach error", "Volume is already exclusively attached to one node and can't be attached to another")
+						contextMsg := fmt.Sprintf("attached node %v", nodes)
+						simpleMsg, detailedMsg := volumeToAttach.GenerateMsg("Multi-Attach error", "Volume is already exclusively attached to one node and can't be attached to another", contextMsg)
 						for _, pod := range volumeToAttach.ScheduledPods {
```

Comment:
Can you not simply concat "attached node %v" message to original `suffixMsg` string? Yes - it would be one more step to creation the full string but it doesn't seem all that different.